source/bringauto/modules/ModuleHandler.cpp

```diff
@@ -22,13 +22,13 @@ void ModuleHandler::destroy() {
 	settings::Logger::logInfo("Module handler stopped");
 }
 
-void ModuleHandler::run() {
+void ModuleHandler::run() const {
 	settings::Logger::logInfo("Module handler started, constants used: queue_timeout_length: {}, status_aggregation_timeout: {}",
 				 settings::queue_timeout_length.count(), settings::status_aggregation_timeout.count());
 	handleMessages();
 }
 
-void ModuleHandler::handleMessages() {
+void ModuleHandler::handleMessages() const {
```

Comment:
There are a lot of functions that are constant. Are you sure all of these const is needed?